### DISCUSSION TITLE
How do I determine the correct GPU ID to use?

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#5172] [Help: Other Questions] [answered]
Version: spacy 2.2.4
Feature: train command
OS: Windows 10

How do I know which value to use for the `--use-gpu, -g` parameter?
Is there a way to list the available gpu and their ID?

**Comment by adrianeboyd** [comment, marked as answer]
This is all configured outside of spacy. If you have one GPU, it's normally `0` and you can see a list of GPUs with `nvidia-smi`. Beyond that, especially if it's a shared server, you may need to contact a sys admin to figure out the details.
